packages/rewards/ISSUES.md

```diff
@@ -0,0 +1,79 @@
+# Analysis
+
+This document outlines all potential issues with the `@recallnet/rewards` package implementation.
+
+## Issues
+
+### 1. **Mathematical Formula Edge Cases**
+
+- **Issue**: The decay formula `w_i = ((1-r) * r^(i-1)) / (1-r^k)` can have numerical instability
+- **Risk**: When `r` approaches 1 or 0, or when `k` is very large, the formula can produce unexpected results
+- **Impact**: Could break reward distribution entirely
+- **Mitigation**: Add bounds checking and fallback calculations
+
+### 2. **Tie Handling**
+
+- **Issue**: No logic for handling tied competitors
+- **Risk**: Undefined behavior when competitors have equal scores
+- **Impact**: Unpredictable results
+- **Mitigation**: Define tie-breaking rules
+
+### 3. **Reward Distribution Completeness**
```

Comment:
If no one boosts a competitor, we may redistribute its prize pool to other competitors. I did no implement this because it's more complex. But def an improvement